library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -302,10 +299,10 @@ class Client() {
         account: SigningKey?,
         appContext: Context?,
         privateKeyBundleV1: PrivateKeyBundleV1,
-        legacyIdentitySource: LegacyIdentitySource,
-        accountAddress: String,
+        address: String,
     ): Pair<FfiXmtpClient?, String> {
         var dbPath = ""
+        val accountAddress = address.lowercase()
```

Comment:
@neekolas I still think there is probably an improvement on the libxmtp side for normalizing addresses maybe in the `getInboxIdForAddress`

Reply:
For sure